modules/nodes-cluster-features-about.adoc

```diff
@@ -1,14 +1,25 @@
 // Module included in the following assemblies:
 //
-// * nodes/nodes-cluster-disabling-features.adoc
+// * nodes/nodes-cluster-enabling-features.adoc
 
 [id="nodes-cluster-features-about_{context}"]
 = Understanding FeatureGates and Technology Preview features
 
-You can use the FeatureGates Custom Resource to toggle on and off Technology Preview features throughout your cluster.
+You can use the FeatureGates Custom Resource to enable Technology Preview
+features throughout your cluster. This allows you, for example, to enable
+Technology Preview features on test clusters where you can fully test them while
+ensuring they are disabled on production clusters.
 
-This allows you, for example, to ensure that Technology Preview features are off for production clusters while leaving the features on for test clusters where you can
-fully test them.
+After turning Technology Preview features on using FeatureGates, they cannot be
```

Comment:
This is very important.  I'm not super familiar with the doc layout, but is there a way to put a gold star on this or something?

Reply:
Yeah, there's an [IMPORTANT] admonition around similar during the procedure itself below, but I'll add it to this block here too.

Reply:
OK, now it appears at the beginning of the [IMPORTANT] block that was right next to it re: support scope:

http://file.rdu.redhat.com/~adellape/061019/oneway_featuregate/nodes/clusters/nodes-cluster-enabling-features.html#nodes-cluster-features-about_nodes-cluster-enabling